src/strategies/index.ts

```diff
@@ -245,7 +246,8 @@ const strategies = {
   'tomb-finance': tombFinance,
   'tranche-staking-slice': trancheStakingSLICE,
   'unipool-same-token': unipoolSameToken,
-  'unipool-univ2-lp': unipoolUniv2Lp
+  'unipool-univ2-lp': unipoolUniv2Lp,
+  'poap-with-weights': poapWithWeights
```

Comment:
Hi, sorry for the late notice, i thought test cases are failing because of master branch changes, but it is actually failing because of this. The strategy folder name is `poap-with-weight` and strategy name mentioned in this file is `poap-with-weight`,

Make it consistent in both :) then we should be good to go